ubuntudesign/snapcraft.io
# Pre sprint changes

## Done

- Changed copy and code on homepage as per copy doc [1]
- Added community page as per copy doc [2]
- Integrated styling for /docs (still needs work)

## QA

Run code, check that page generates and loads. Cross ref any issues with Issue queue.

## Issue / Card

[1] Homepage copy doc: https://docs.google.com/document/d/1HPW-eJsRLJcH6Z58F8ghMQA2EpeSTMZ-kIsQDj0e8wA/edit

[2] https://docs.google.com/document/d/1WPH9yIFZX6ML8sVGOd90EkJvQ9XplyTOwn6-Ad26zNw/edit

[3] Integrated /docs
